libraries/AP_MSP/AP_MSP_Telem_Backend.cpp

```diff
@@ -618,13 +618,13 @@ MSPCommandResult AP_MSP_Telem_Backend::msp_process_out_comp_gps(sbuf_t *dst)
         uint16_t dist_home_m;
         uint16_t home_angle_deg;
         uint8_t toggle_gps;
-    } p;
+    } gps;
 
-    p.dist_home_m = home_state.home_distance_m;
-    p.home_angle_deg = home_angle_deg;
-    p.toggle_gps = 1;
+    gps.dist_home_m = home_state.home_distance_m;
```

Comment:
This is a candidate for brace-initialisation like we do a lot of in AP_Logger:  https://github.com/ardupilot/ardupilot/blob/master/libraries/AP_Logger/LogFile.cpp#L225

Reply:
ok